scripts/submit_emod3d.py

```diff
@@ -60,6 +60,19 @@ def main(args: argparse.Namespace, est_model: est.EstModel = None, logger: Logge
             model,
             True,
         )
+        # scale up the est_run_time if it is a re-run (with check-pointing)
```

Comment:
Why only scale up if checkpointing files exist?

Reply:
otherwise its either an 'on purpose' cancel or something is so wrong that it might not even be restart-able